.spectral.yaml

```diff
@@ -1,2 +1,6 @@
 extends: '@rhoas/spectral-ruleset'
+rules:
+  no-$ref-siblings: off
```

Comment:
what was the reason this was disabled?

Reply:
it is no longer a thing in OAS v3.1
They've completely got rid of it. May have something to do with the json schema version support but I'm not 100%, either way it doesn't appear to have any effect. 
@wtrocki may be able to give a better explanation

Reply:
Instead of explanation I will come up with question. Can we update your openapi version to 3.1? From what I see it would not affect much

Reply:
I don't see why not :thinking: can you confirm @machi1990?

Thanks for taking on this work :)

Reply:
@wtrocki I was going to bump up the API versions across the board but I think they should be in separate PRs, they may have breaking changes etc. even if not, they should be seperated into their own PRs imho

Reply:
agreed for bumping it in separate PR. Thanks for the explanation! :)

Reply:
> I don't see why not thinking can you confirm @machi1990?

It should be okay but as already suggested maybe something to do in a separate PR.